### ISSUE TITLE
Does not work on Debian testing

### ISSUE THREAD

**Issue by aleksejrs** [jepler/cropgui#55]
My existing package of cropgui stopped working recently, saying:

```
Traceback (most recent call last):
  File "/usr/bin/cropgui", line 22, in <module>
    import ImageTk
ImportError: No module named ImageTk
```

I installed a package made from https://github.com/jepler/cropgui/commit/a6e9388751920875a9b12a57fd3b3e92d77019c6 using `checkinstall ./install.sh -p /usr -P /usr/bin/python`.  The result when running cropgui:

```
Traceback (most recent call last):
  File "/usr/bin/cropgui", line 25, in <module>
    import filechooser
ImportError: No module named filechooser
```

If I build with `-f tk`, the former error appears.

**Comment by aleksejrs** [comment]
The error was from a missing `python-imaging` package (since I generated the cropgui package using checkinstall, it had no dependencies.)

I checked that all the packages listed in the README for Debian are installed. Now it says "ImportError: No module named log".

I have problems making a working package. If I run "debuild", the executable file is owned by the user who runs it, and has -rwxr-x--x permissions.

**Comment by drzraf** [comment]
... nor on Ubuntu 18.04

**Comment by aleksejrs** [comment]
It works for me currently, at least with some pictures.

**Comment by MattBlissett** [comment]
I made this change so cropgui would work on Ubuntu 18.04:

```diff
@@ -19,7 +19,7 @@ from cropgui_common import *
 from cropgui_common import _
 
 import Tkinter
-import ImageTk
+from PIL import ImageTk
 import tkFileDialog
 import sys
 import os
```

**Comment by muralikodali** [comment]
tested on ubuntu 18.04.01 . not working.

**Comment by Frenzie** [comment]
What do you mean by "not working"? I just tried it and if you install `python-pil` (which apt tells you replaces the python-imaging packages ;-) ) then cropgtk.py loads. Slightly trickier perhaps is that you also need `python-pil.imagetk`, but apt does that automatically for you.

tl;dr I think this should do it:

```
sudo apt install python python-pil python-pil.imagetk libjpeg-progs libimage-exiftool-perl
```

**Comment by muralikodali** [comment]
tried on your suggestion . but no use.

following is displayed on terminal, when i tried to execute cropgui script:

Traceback (most recent call last):
  File "/home/murali/bin/cropgui.sh", line 19, in <module>
    import Image
ImportError: No module named Image

**Comment by Frenzie** [comment]
Is it saying "No module named Image" or no module named Image*Tk*? You will need the necessary packages as well as the source mod from https://github.com/jepler/cropgui/issues/55#issuecomment-390891750

**Comment by kanliot** [comment]
Thanks.  Got it working on Debian 10 "buster"

**Comment by jepler** [comment]
Closing as I believe this is outdated. cropgui and cropgtk work on my debian 12 machine.
